### PR TITLE
INSTUI-3224 - add theme types to components

### PR DESCRIPTION
**Pull request by ke1k0**

### CODE REVIEW

**Review by matyasf — APPROVED** [review]
looks good from what I've tested

File: packages/ui-drawer-layout/src/DrawerLayout/DrawerContent/styles.ts [hunk 1]
```diff
@@ -32,8 +34,13 @@
...
  * @return {Object} The final style object, which will be used in the component
  */
-// @ts-expect-error ts-migrate(6133) FIXME: 'props' is declared but its value is never read.
-const generateStyle = (componentTheme: any, props: any, state: any) => {
+const generateStyle = (
+  componentTheme: DrawerLayoutContentTheme,
+  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any'... Remove this comment to see the full error message
+  props,
+  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'state' implicitly has an 'any'... Remove this comment to see the full error message
+  state
```
[the remark this answers is not in the document]
Comment (ke1k0) [reply packages/ui-drawer-layout/src/DrawerLayout/DrawerContent/styles.ts:42]: Either that or that's the next PR I was planning to do after this one :)

File: packages/ui-view/src/View/theme.ts [hunk 2]
```diff
@@ -46,67 +56,77 @@ const generateComponentTheme = (theme) => {
...
+  console.log(asd1)
+  // eslint-disable-next-line no-console
+  console.log(asd2)
+  // eslint-disable-next-line no-console
+  console.log(asd3)
+  // eslint-disable-next-line no-console
+  console.log(asd4)
+  // eslint-disable-next-line no-console
+  console.log(asd5)
+
```
Comment (Brailor) [inline packages/ui-view/src/View/theme.ts:127]: I think we can delete these :)

Comment (ke1k0) [reply packages/ui-view/src/View/theme.ts]: Oops, sorry, deleted! Thanks :D

**Review by Brailor — APPROVED** [review]

**Pull request merged**